.github/workflows-config/aws-openshift-rosa-hcp-single-region/test_matrix.yml

```diff
@@ -0,0 +1,42 @@
+---
+matrix:
+    distro:
+        # /!\ BEFORE ADDING/REMOVING A VERSION:
+        # /!\ Please keep this matrix synced with the official documentation:
+        # https://github.com/camunda/camunda-docs/blob/main/docs/self-managed/setup/deploy/openshift/redhat-openshift.md?plain=1#L2
+        # According to https://access.redhat.com/support/policy/updates/openshift, this matrix should reference the last 4 (may change) supported versions of OpenShift
+        - name: OpenShift 4.18
+          type: openshift
+          # renovate: datasource=custom.rosa-camunda depName=red-hat-openshift versioning=regex:^4(\.(?<minor>\d+))?(\.(?<patch>\d+))?$
+          version: 4.18.1
+          # /!\ Warning: When a new minor version of OpenShift is released,
+          # you must update all N-1, N-2, and N-3 versions in this matrix.
+          # rationale: Red Hat supports the last four minor versions of OpenShift.
+          # Therefore, to ensure compatibility and support, we must test against these versions.
+          # For more details, refer to the official support policy at https://endoflife.date/red-hat-openshift.
+          platform: rosa
+          # According to https://access.redhat.com/support/policy/updates/openshift,
+          # this matrix should reference the last 4 (may change) supported versions of OpenShift
+
+        - name: OpenShift 4.17
+          type: openshift
+          # renovate: datasource=custom.rosa-camunda depName=red-hat-openshift versioning=regex:^4.17(\.(?<patch>\d+))?$
+          version: 4.17.16
+          platform: rosa
+
+        - name: OpenShift 4.16
+          type: openshift
+          # renovate: datasource=custom.rosa-camunda depName=red-hat-openshift versioning=regex:^4.16(\.(?<patch>\d+))?$
+          version: 4.16.8
+          platform: rosa
+
+        - name: OpenShift 4.15
+          type: openshift
+          # renovate: datasource=custom.rosa-camunda depName=red-hat-openshift versioning=regex:^4.15(\.(?<patch>\d+))?$
+          version: 4.15.21
+          platform: rosa
```

Comment:
just my opinion but would either go with lowest supported or highest supported but not all versions.
Let's see what the QA meeting brings tomorrow but atm don't see us as team ensuring that supported envs are actually followed.